From ba6a035c90562f31c78df3b5bf8447182c87a812 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 20:50:13 +0900
Subject: [PATCH 520/909] arm64: dts: r8a77990-ebisu: Add no use ddc option for
 VGA connector

commit 544b1bf429845ef319b9f92bac879b29e1c42440 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

By setting this option, change to not use DDC (Display Data Channel)
on the VGA connector.

Based on a similar patch of the Salvator common device tree
by Koji Matsuoka <koji.matsuoka.xm@renesas.com>.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 3dd3c61c3621..4854ab38cfff 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -168,6 +168,8 @@
 	vga {
 		compatible = "vga-connector";
 
+		no-use-ddc;
+
 		port {
 			vga_in: endpoint {
 				remote-endpoint = <&adv7123_out>;
-- 
2.17.1

